jruby-gradle-base-plugin/embedded-rubygems-server/build.gradle

```diff
@@ -0,0 +1,35 @@
+import com.github.jrubygradle.JRubyExec
```

Comment:
Is this `build.gradle. supposed to be a gradleTest example case, or is it a separate build project for the multiproject build.

If it's the latter, then I'd prefer to move it to the top level directory so we have all our child projects at the same level in the tree.